index.html

```diff
@@ -10,11 +10,13 @@
   </head>
   <body>
     <header>
-      <h1 id="logo">Aziz Mejri</h1>
-      <button type="button" class="menu-bars">
-        <i class="fas fa-bars"></i>
-      </button>
-      <nav id="desktop-nav">
+      <div id="mobile-nav">
+        <h1 id="logo">Aziz Mejri</h1>
```

Comment:
- [ ] Upon clicking the hamburger button, kindly make this logo hidden by adding `display: none;` then should do otherwise upon another click.